### PR TITLE
Don't shutdown the notifications when it freezes by browser shutdown

### PR DESCRIPTION
**Pull request by nickvergessen**
Another attempt to try to recover notifications when the browser reopens after a night of suspend.

### CODE REVIEW

**Comment by rullzer** [comment]
sure...

@ChristophWurst I guess we should have the notifications also isten to the online/offline event?

**Comment by nickvergessen** [comment]
It is already listening, but in case of 404 we stopped it.
I would have assumed we dont get a 404 in case the browser suspends. So this is another attempt towards this.

**Comment by rullzer** [comment]
> It is already listening, but in case of 404 we stopped it.
> I would have assumed we dont get a 404 in case the browser suspends. So this is another attempt towards this.

AH fair enough

File: src/App.vue [hunk 1]
```diff
@@ -237,8 +237,6 @@ export default {
 					} else if (err.response.status === 404) {
 						// 404 - App disabled
 						console.info('Shutting down notifications: app is disabled.')
```
Comment (rullzer) [inline src/App.vue:239]: This is no longer correct now right?

Comment (nickvergessen) [reply src/App.vue]: Yeah, fixed

**Pull request merged**